output-tests/draft2019-09/output-schema.json

```diff
@@ -44,14 +41,7 @@
             }
           },
           "then": {
-            "anyOf": [
-              {
-                "required": [ "error" ]
-              },
-              {
-                "required": [ "errors" ]
-              }
-            ]
+            "required": [ "errors" ]
```

Comment:
This was correct.  The node either needs a string error (`error`) or a collection of error units (`errors`) (or both).